### PR TITLE
Fixes for terminology refactor

### LINKED ISSUE #451: Configuration Dropdown not populating with options.
**Describe the bug**
When in the yap admin if i go to configure a service body the Configuration Dropdown is not populating with options. I tested this on SEZF and NERNA servers running 3.8.1

**What version are you running?**
3.8.1

**Did this happen on prior versions to the one you are running?**
not sure about 3.8.0 but it def didn't happen on 3.7.5.

**To Reproduce**
In the yap admin, go to configure a service body and try to add a configuration option.

**Expected behavior**
The configuration options are available to edit.

**Screenshots**
Screen Shot 2020-07-25 at 12 35 50 PM

**Additional context**
If setting were set previously they still show up and work, just are not able to set new ones.

### PR DESCRIPTION
**Pull request by pjaudiomv**
resolves #451

### CODE REVIEW

File: RELEASENOTES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+### 3.8.2 (UNRELEASED)
```
Comment (radius314) [inline RELEASENOTES.md:3]: will release right away, bump the date

Comment (pjaudiomv) [reply RELEASENOTES.md]: bumped

Comment (radius314) [reply RELEASENOTES.md]: bump the version in functions.php as well 🐈

File: RELEASENOTES.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+### 3.8.2 (July 25, 2020)
+* Fixes for terminology refractor. [#451]
```
Comment (radius314) [inline RELEASENOTES.md:4]: typo: `refractor` to `refactor`

Comment (pjaudiomv) [reply RELEASENOTES.md]: oops

File: RELEASENOTES.md (hunk 1 above)
Comment (pjaudiomv) [reply RELEASENOTES.md]: good idear

**Pull request merged**